### PR TITLE
[DO NOT MERGE, PLS] Docker fixes

### PR DESCRIPTION
**Pull request by konstantinzolotarev**

### CODE REVIEW

**Review by jar-o — APPROVED** [review]

File: systemd/docker-omnia.service [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Dockerised Omnia
+Requires=docker.service
+After=docker.service
+
+[Service]
+Type=oneshot
```
Comment (teghnet) [inline systemd/docker-omnia.service:7]:
```suggestion
Type=oneshot
```

File: systemd/docker-omnia.service [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+RemainAfterExit=yes
```
Comment (teghnet) [inline systemd/docker-omnia.service:8]: I don't know about this one.

File: systemd/docker-omnia.service [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+User=$USER
+Group=$USER
+WorkingDirectory=/home/terra
+ExecStart=/usr/local/bin/docker-compose up -d
+ExecStop=/usr/local/bin/docker-compose down
+TimeoutStartSec=0
+RestartSec=30
```
Comment (teghnet) [inline systemd/docker-omnia.service:15]:
```suggestion
RestartSec=5
Restart=always
```

File: systemd/docker-omnia.service (hunk 1 above)
Comment (konstantinzolotarev) [reply systemd/docker-omnia.service:7]: Removing this line will endup in always `dead` service

**Pull request closed without merging**